flixel/input/gamepad/FlxGamepad.hx

```diff
@@ -154,6 +203,11 @@ class FlxGamepad implements IFlxDestroyable
 		return gamepadButton;
 	}
 	
+	public inline function getFlipAxis(AxisID:Int):Int
```

Comment:
Could this be used to invert the y axis of the Xbox 360 controller in chrome, which is currently hardcoded in getYAxis?